labs/gps_lab/README.md

```diff
@@ -0,0 +1,68 @@
+#GPS LAB
+
+##Description
+GPS Lab uses a Raspberry PI with a GPS sensor to fetch current location (latitude and longitude)
+ and sends the info to IoT hub.
+
+ A web application reads the data from IoT Hub real time and displays the GPS location in a Azure MAP.
+
+ ##Steps
+ ### Create and configure IoT Hub
+ 1. Create one azure IoT Hub and create one device under it 
+ 2. Install azure CLI (https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+ 3. Add a consumer group to your IoT hub (Using azure CLI OR from Azure Portal)
+       
+        az iot hub consumer-group create --hub-name YourIoTHubName --name YourConsumerGroupName
```

Comment:
Can this be in a shell code block:

```sh
az iot hub consumer-group create --hub-name YourIoTHubName --name YourConsumerGroupName
```

Can you also explain the parameters that the users have to set.

Same for all other commands the users have to run